core/retries-api/src/main/java/software/amazon/awssdk/retriesapi/AcquireInitialTokenRequest.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.retriesapi;
```

Comment:
Should it be `software.amazon.awssdk.retries.api`?

Reply:
I just realized that the original plan was to use instead `software.amazon.awssdk.api.retries`, I can go either way but I like the original idea better, thoughts?